src/patternfly/components/Menu/menu.scss

```diff
@@ -724,7 +725,8 @@
     --#{$menu}__item-action--Color: var(--#{$menu}__item-action--hover--Color);
   }
 
-  &:disabled {
+  &:disabled,
+  &.pf-m-aria-disabled {
```

Comment:
@thatblindgeye good callout. It would be more consistent, and simplify the way you disable a menu item. I think the only benefit is that if the menu item is a `<button>`, it will have to have `disabled`, and that can trigger the styling without the need to also add a class.